nvflare/apis/impl/list_resource_manager.py

```diff
@@ -55,7 +56,7 @@ def check_resources(self, resource_requirement: dict, fl_ctx: FLContext) -> (boo
                 if len(self.resources[k]) < resource_requirement[k]:
                     check_result = False
                     break
-        return check_result, None
+        return check_result,  str(uuid.uuid4())
```

Comment:
we don't need this token in this `ListResourceManager`.

This token would be needed on the ResourceManager that reserves certain resources upon checking, so they need the token to cancel those reserved resources if the job is not deployed.

Reply:
resource manager need to return a token back to the allocated resources. The allocated resources are used by the resource consumer. Resource consumer needs to check this token matching. This logic is missing in the resource consumer right now.

Reply:
The original "token" here is designed for reserve / unreserve resources (like if requesting some cloud resources)
@yanchengnv can correct me if I am wrong.

In this implementation, we are not reserving any resources when we do check_resource.
Thus, the token is unused in this implementation.

Why do we need another token in resource consumer?